app/Actions/LamboAction.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Actions;
+
+use Exception;
+
+trait LamboAction
+{
+    protected function line(string $message)
```

Comment:
Can we pull line, info, error, and warn into a new trait and then pull that trait into this one?

Reply:
Done.